src/out-resolve.js

```diff
@@ -1,13 +1,19 @@
 import path from 'path';
 
-export default (input, output) => new Promise(resolve => {
+export default (input, {output, root, allInOutput} = {}) => new Promise(resolve => {
```

Comment:
[memo]
If you don't pass an argument, destructuring assignment can't be performed and will result in an error, so the default value is set.

Reply:
Yes I understand, this is a good improvement